lib/tasks/vmodl_helper.rb

```diff
@@ -149,14 +149,18 @@ def dump_vmodl!
   private
 
   def build_wsdl_obj!(type)
+    type_name = type.name.name
```

Comment:
There were no new object types in the 7.0U3f vim.wsdl so I didn't hit this when running https://github.com/ManageIQ/rbvmomi2/pull/16 after extracting this to a method :cry: